crates/core/database/src/models/server_members/ops/mongodb.rs

```diff
@@ -238,11 +312,25 @@ impl AbstractServerMembers for MongoDb {
         )
         .map(|_| ())
     }
+
+    async fn remove_dangling_members(&self) -> Result<()> {
+        let now = Timestamp::now_utc();
+        let date = format!("{}", now.format_short()); // I hate this library
+
+        self.col::<Document>(COL)
+            .delete_many(doc! {
+                "pending_deletion_at": {"$lt": date}
```

Comment:
I think the preferred strategy here would be to do `to_bson(Timestamp::now_utc())`